README.md

```diff
@@ -1,12 +1,12 @@
 # ![Shaka Player](docs/shaka-player-logo.png)
 
-Shaka Player is an open-source JavaScript library for adaptive media.  It plays
+Shaka Player is an open-source JavaScript library for adaptive media.  It plays
 adaptive media formats (such as [DASH][] and [HLS][]) in a browser, without
-using plugins or Flash.  Instead, Shaka Player uses the open web standards
+using plugins or Flash.  Instead, Shaka Player uses open web standards
 [MediaSource Extensions][] and [Encrypted Media Extensions][].
 
-Shaka Player also supports [offline storage and playback][] of media using
-[IndexedDB][].  Content can be stored on any browser.  Storage of licenses
+Shaka Player also supports [offline storage and playback][] of media use
+[IndexedDB][].  Content can be stored on any browser.  The storage of licenses
```

Comment:
correcting use & the storage

Reply:
"Storage" is a valid subject in English without the need for "the", but "The storage of licenses" arguably sounds a little better.  So that can stay.

The other change just above should be reverted.  "Shaka supports storage of media **using** IDB" is correct.  "Shaka supports storage of media **use** IDB" does not make sense in English.